src/applications/edu-benefits/0994/containers/IntroductionPage.jsx

```diff
@@ -17,15 +50,34 @@ export class IntroductionPage extends React.Component {
           Equal to VA Form 22-0994 Application for Veteran Employment Through
           Technology Education Courses (VET TEC)
         </p>
-        <SaveInProgressIntro
-          startMessageOnly
-          prefillEnabled={this.props.route.formConfig.prefillEnabled}
-          verifyRequiredPrefill={
-            this.props.route.formConfig.verifyRequiredPrefill
-          }
-          messages={this.props.route.formConfig.savedFormMessages}
-          pageList={this.props.route.pageList}
-        />
+        {isBrandConsolidationEnabled() ? (
```

Comment:
I think we can remove `IsBrandConsolidationEnabled`

Reply:
Done!